### DISCUSSION TITLE
How to remove unused auth methods from the login screen

### DISCUSSION THREAD

**Discussion by msaeedm** [outline/outline#3859] [Self Hosting] [answered]
Hello,

I setup Outline to use my self-hosted Authelia for authentication which is working well. I would like to remove the Slack (default) auth option from the login screen so the login screen would only contain the OpenID option. How do I do that? I tried removing slack options from the env file but that did not make a difference.

Thank you,

**Comment by msaeedm** [comment, marked as answer]
The slack option went away now. Guess it was a cache thing in the browser.

**Reply by yoramshai** [reply]
@msaeedm, after removing the slack auth option, could you keep slack integration, or must the auth and the slack integration come together?
